tests/README.md

```diff
@@ -18,19 +22,7 @@ First start your application with Speculos
 then in the `tests` folder run
 
 ```
-pytest
-```
-
-### Launch with your Nano S/X
```

Comment:
No need for this anymore?

Reply:
The integration tests were meant to run with either speculos (a ledger emulator via http api) or with your own ledger device and the ledgercomm with hid dependency was meant to be the transport layer, this was taken from the documentation at the time but this dependency was not working at all.
So I implemented the HTTP api for speculos but could not implement the hid transport, maybe this was fixed on a newer version and we can revisit this idea but for now I think we should remove it.